### PR TITLE
feat(config): allow to set the configuration path from `WTF_CONFIG`

### PR DESCRIPTION
**Pull request by noirbizarre**
Hello 👋🏼 

This PR is just adding the possibility to set the configuration file path from the `WTF_CONFIG` environment variable (allowing use cases like by directory dashboard using `direnv`)

### CODE REVIEW

**Comment by senorprogrammer** [comment]
I like this, good idea.

**Comment by noirbizarre** [comment]
Hi 👋🏼 

Is there something preventing to merge this one ?
As you seem to like this one, if there is any change or improvement needed, I would gladly make them ASAP so you can merge it

**Comment by senorprogrammer** [comment]
Nope, nothing needs to be changed, life just got in the way. Apologies for the long delay.

File: flags/flags.go [hunk 1]
```diff
@@ -161,11 +161,18 @@ func (flags *Flags) Parse() {
...
 		fmt.Printf("Error: %v\n", err)
 		os.Exit(1)
 	}
-	flags.Config = filepath.Join(configDir, "config.yml")
+	envCfg := os.Getenv("WTF_CONFIG")
+	if envCfg == "" {
+		flags.Config = envCfg
+	} else {
+		flags.Config = filepath.Join(configDir, "config.yml")
+	}
```
Comment (senorprogrammer) [inline flags/flags.go:177]: This looks strange to me - if the os ENV var is empty, it sets `flags.Config` to it, which would make that empty. If it's not empty then it uses the file path?

Comment (noirbizarre) [reply flags/flags.go:177]: My bad, it swapped the statements. Fixed

**Comment by noirbizarre** [comment]
> Nope, nothing needs to be changed, life just got in the way. Apologies for the long delay.

No worries, kind of the same for me.

Anyway, I fixed the swapped statements, it should be OK

**Comment by senorprogrammer** [comment]
Thanks for making this improvement, this is excellent.

**Pull request merged**